tcl/org.testeditor.tcl.model/src/org/testeditor/tcl/util/TclModelUtil.xtend

```diff
@@ -210,9 +210,12 @@ class TclModelUtil extends TslModelUtil {
 
 	def ValueSpaceAssignment getValueSpaceAssignment(Component component, TestStep container) {
 		for (element : component.elements) {
-			val valueSpace = getValueSpaceAssignment(element, container)
-			if (valueSpace != null) {
-				return valueSpace
+			val stepContentElementsStrings = container.contents.filter(StepContentElement).map[value]
```

Comment:
I don't understand this change. If it didn't work, shouldn't the method `getValueSpaceAssignment(ComponentElement element, TestStep container)` be fixed? It would also be nice if we had a test in `TclModelUtilTest`.